test/util/textEditorFactory.js

```diff
@@ -10,6 +10,16 @@ module.exports = function textEditorFactory (input = {}) {
     getText () {
       return input.source || ''
     },
+    getBuffer () {
+      return {
+        positionForCharacterIndex (x) {
+          // This method is supposed to return the position as an array
+          // [line, column] as zero indexed numbers. For now it is just stubbed
+          // out.
+          return [0, 0]
+        }
+      }
+    },
```

Comment:
I'm not quite satisfied with this. It's necessary to map between the fixes suggested by eslint and the position in the `textBuffer`. eslint reports positions as character indexes, and luckily there's a method to map that to the position tuples builtin to atom.

It's a quite crude stub, but we aren't really supposed to be testing the logic in eslint anyway, so it might be fine.